quinn-proto/src/transport_parameters.rs

```diff
@@ -139,7 +139,6 @@ impl TransportParameters {
                     .try_into()
                     .expect("setter guarantees this is in-bounds")
             }),
-            max_ack_delay: 0u32.into(),
```

Comment:
Is there any reason to change this before we actually start sending nonzero values?

Reply:
It doesn't really make a change for this anymore, and is mostly a leftover from the version where the commit reported actual delay. I decided to keep it here to minimize the size of the next change, and since it doesn't really make a difference on where it is - as long as no delay is reported the situation is the same for the peer anyway.

Plus it was kind of weird that that this field is explicitely set to something different than the default value :-)

Reply:
I figured it might be useful to a peer, but realistically it's probably moot.